packages/react/test/specs/components/Accordion/Accordion-test.tsx

```diff
@@ -0,0 +1,227 @@
+import * as React from 'react'
+import * as keyboardKey from 'keyboard-key'
+
+import Accordion from 'src/components/Accordion/Accordion'
+import { isConformant, handlesAccessibility } from 'test/specs/commonTests'
+import { mountWithProvider, mountWithProviderAndGetComponent } from 'test/utils'
+import AccordionTitle from 'src/components/Accordion/AccordionTitle'
+import { ReactWrapper, CommonWrapper } from 'enzyme'
+
+const panels = [
+  {
+    key: 'one',
+    title: 'One',
+    content: '2 3 4',
+  },
+  {
+    key: 'two',
+    title: 'Five',
+    content: '6 7 8 9',
+  },
+  {
+    key: 'three',
+    title: "What's next?",
+    content: '10',
+  },
+]
+
+const getTitleButtonAtIndex = (wrapper: ReactWrapper, index: number): CommonWrapper => {
+  return wrapper
+    .find(`.${AccordionTitle.slotClassNames.button}`)
+    .filterWhere(n => typeof n.type() === 'string')
+    .at(index)
+}
+
+describe('Accordion', () => {
+  isConformant(Accordion)
+
+  describe('activeIndex', () => {
```

Comment:
Very good tests, all expected behavior is nicely describe trough the tests. Very good job! 🥇